main.py

```diff
@@ -399,20 +398,19 @@ def main():
                                 # order not filled, try again.
                                 logger.info(f"clearing order with a status of {order_status}.  Waiting for 'closed' status")
                                 order.clear()  # reset for next iteration
-                        
-                            
                     else:
                         logger.warning(f'{announcement_coin=} is not supported on gate io')
                         if os.path.isfile('new_listing.json'):
                             os.remove("new_listing.json")
+                        old_coins.append(announcement_coin)
                         logger.debug('Removed new_listing.json due to coin not being '
                                     'listed on gate io')
                 else:
                     logger.error('supported_currencies is not initialized')
             else:
                 logger.info( 'No coins announced, or coin has already been bought/sold. Checking more frequently in case TP and SL need updating')
 
-            time.sleep(1)
+            time.sleep(3)
```

Comment:
Increased this again, since I and at least one other user have experienced issues with a too low sleep time. We can optimize this later, I would say, once we know everything is stable.